pkgs/by-name/cm/cmake/005-remove-systemconfiguration-dep.diff

```diff
@@ -1,10 +1,10 @@
 diff -Naur cmake-3.27.7/Utilities/cmcurl/CMakeLists.txt cmake-3.27.7-new/Utilities/cmcurl/CMakeLists.txt
 --- cmake-3.27.7/Utilities/cmcurl/CMakeLists.txt	2023-10-06 10:08:35.000000000 -0300
```

Comment:
I changed the diffs manually, so it means those headers didn't change. Should I regenerate from scratch or hand written diffs are fine?

Reply:
I personally prefer them generated via `diff old-dir new-dir`.

Reply:
Done! I actually edited the CMake source and ran `diff -Naur cmake-3.28.1/... cmake-3.28.1-new/...` again to get this output